app/src/main/java/com/platform/APIClient.java

```diff
@@ -148,16 +151,20 @@ public Response sendRequest(Request locRequest, boolean needsAuth, int retryCoun
             byte[] decompressed = gZipExtract(data);
             postReqBody = ResponseBody.create(null, decompressed);
             try {
-                Timber.d("timber: sendRequest: (%s)%s, code (%d), mess (%s), body (%s)", request.method(),
-                        request.url(), response.code(), response.message(), new String(decompressed, "utf-8"));
+                if (response.code() != 200) {
+                    Timber.d("timber: sendRequest: (%s)%s, code (%d), mess (%s), body (%s)", request.method(),
+                            request.url(), response.code(), response.message(), new String(decompressed, "utf-8"));
+                }
             } catch (UnsupportedEncodingException e) {
                 Timber.e(e);
             }
             return response.newBuilder().body(postReqBody).build();
         } else {
             try {
-                Timber.d("timber: sendRequest: (%s)%s, code (%d), mess (%s), body (%s)", request.method(),
-                        request.url(), response.code(), response.message(), new String(data, "utf-8"));
+                if (response.code() != 200) {
```

Comment:
reduce logging...otherwise it was reading the whole Pricing JSON

Reply:
Awesome!